lib/axp209.js

```diff
@@ -85,11 +96,38 @@ AXP209.prototype._readButton = function() {
   return (value ? 1 : 0);
 };
 
+AXP209.prototype._readVcc18Adc = function() {
+  // enable 12-bit ADC for VCC 1.8 
+  var p = this._i2c.readRegister(0x85, 1)[0]; // 0-2v range or .7 - 2.7v range?
+  var rhigh = this._i2c.readRegister(0x64, 1)[0].toString(16);
+  var rlow = this._i2c.readRegister(0x65, 1)[0].toString(16);
+  var r = parseInt(rhigh + rlow, 16);
+  
+  var voltage = (((r * 10000)/4096) * 2) + (p * 7000); // voltage in millivolts
+  
+  return voltage;
+};
+
 AXP209.prototype._configureBatAdc = function() {
   // force ADC enable for battery voltage and current
   return this._i2c.writeRegister(BAT_ADC_REGISTER, new Buffer([0xc3]));
 };
 
+AXP209.prototype._configureVcc18Adc = function() {
+  // force enable 12-bit ADC for VCC 1.8 
+  this._i2c.writeRegister(0x83, new Buffer([0x80])); // disable ADC input on GPIO0
+  this._i2c.writeRegister(0x90, new Buffer([0x04])); // set GPIO0 to 12 bit ADC input
+  this._i2c.writeRegister(0x85, new Buffer([0x00])); // use 0x01 for ADC input range 0.7-2.7475v
+  this._i2c.writeRegister(0x83, new Buffer([0x88])); // enable ADC input on GPIO0
+};
+
+AXP209.prototype._resetVcc18 = function() {
```

Comment:
you're right that's a good idea.  I wasn't sure where we could put this,  but there should be a way for the user to reset the VCC-1V8 pin in case they want to use it as a 1.8 V voltage source again.